docs/quickstart.md

```diff
@@ -0,0 +1,411 @@
+# Quickstart guide
+
+## Prerequisites
+
+1. Learn the [basic features](index.md).
```

Comment:
> Learning the basic features is less of a prerequisite than having an OpenAPI definition they can use to actually try the tool.
Yes, agreed. We can actually leave 2 points here - installation and a yaml file to try out

> Instead of pasting the entire contents of an example definition into the quickstart guide in the next section, just add a link to the YAML file here, in the Prerequisites section. Link to this one from the developer-portal-starter, for example.

Can't agree 100%. Pasting a link is good from the unification perspective but I think giving a link to something that can be changed or moved is a bad idea. What if the content will be changed tomorrow? This guide will become irrelevant immediately and you will be forced to update it as soon as possible to avoid negative feedback or complaints about non-working or irrelevant steps.

 It's a user journey and we are trying to follow them through their very first introduction to the product. They must see and "feel" the process, instead of going here and there to find a particular file. Everything must be under their fingertips even if it requires you to produce some duplication or not-so-ideal steps. This is how the learning process goes. The users must understand the idempotency - If I use this example, I will always receive this result.

Reply:
Okay, thank you for explaining your perspective! I understand now why you decided to present the content in this way. The issue with the content potentially changing tomorrow is valid for any documentation in the world so it's probably not the biggest problem here. But either way, it should be fine if you keep your approach. Maybe just explain a little bit about rules and what makes those errors get detected as errors - I hope you'll agree that information will be useful. :)